### Laelaps laelaps_control/GetProductInfo service
#
# Get the robot's product information.
#

---
ProductInfo i   # product information
